src/components/RegisterLoginHeader/RegisterLoginHeader.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import Switch from '@mui/material/Switch';
+
+import styles from './RegisterLoginHeader.module.css';
+
+function RegisterLoginHeader(props) {
```

Comment:
You could unpack the props right away with destructuring: `{checked, onChange}`.